app/views/users/show.html.erb

```diff
@@ -0,0 +1,54 @@
+
+<h1>Hello from the admin show page!</h1>
+
+This is where one day we will be able to edit a trail, edit a user, delete a trail, or delete a user.
+
+
+<%= form_tag(user_path, method: :get) do |f| %>
+<br />
+<div id="selectUser">
+
+  User To Update: <%= select("user", "username", User.all.collect(&:username), :id => 'user-select') %>
+</div>
+
+  <div class="actions">
+    <%= submit_tag("Search") %>
+  </div>
+
+<% end %>
+
+<% if @user.id %>
+  <br />
+  <br />
+    <%= form_for(@user) do |f| %>
+
+    <div class="field">
```

Comment:
indent one layer in to indicate it is "inside" the `form_for` block initialized above